gensim/test/test_sklearn_api.py

```diff
@@ -177,7 +177,7 @@ def testCSRMatrixConversion(self):
         newmodel.fit(sarr)
         bow = [(0, 1), (1, 2), (2, 0)]
         transformed_vec = newmodel.transform(bow)
-        expected_vec = numpy.array([0.35367903, 0.64632097])
+        expected_vec = numpy.array([0.12843782, 0.87156218])
```

Comment:
Why this change?

Reply:
I think because now this script interprets the input differently -> different model -> different result, but looks suspicious, I agree.

Reply:
@piskvorky As Ivan correctly pointed out, since the input matrix used for training (by `fit` method) is being interpreted differently after making this change, the model changes and thus, the output of `transform` changes as well.